operator/v1/types.go

```diff
@@ -70,11 +70,11 @@ type OperatorSpec struct {
 	// +kubebuilder:default=Normal
 	OperatorLogLevel LogLevel `json:"operatorLogLevel,omitempty"`
 
-	// unsupportedConfigOverrides holds a sparse config that will override any previously set options.  It only needs to be the fields to override
-	// it will end up overlaying in the following order:
-	// 1. hardcoded defaults
-	// 2. observedConfig
-	// 3. unsupportedConfigOverrides
+	// UnsupportedConfigOverrides adds configuration changes that are not supported by an operator.
```

Comment:
Must be lower case `u` on this so that the name matches the json tag, which is what the user would see